src/components/Table/hoc/withTableActions/withTableActions.tsx

```diff
@@ -64,8 +65,10 @@ export type TableActionConfig<I> = TableAction<I> | TableActionGroup<I>;
  */
 export type TableRowActionsSize = 's' | 'm' | 'l' | 'xl';
 
+export type RenderRowActionsProps<I> = {item: I; index: number};
 export interface WithTableActionsProps<I> {
-    getRowActions: (item: I, index: number) => TableActionConfig<I>[];
+    getRowActions?: (item: I, index: number) => TableActionConfig<I>[];
+    renderRowActions?: (props: RenderRowActionsProps<I>) => React.ReactNode;
```

Comment:
May be same interface as a `getRowActions`?
`getRowActions?: (item: I, index: number) => React.ReactNode;`

Reply:
it is for passing React components instead simple functions 

https://github.com/gravity-ui/uikit/pull/1353/files#diff-6f66bd528526ae4a3ec1ed23a7222fdf1de5086ccdd520c5047c2c9f677e57ecR145-R155